### PR TITLE
✨ Add game components #6

### LINKED ISSUE #5: Create play components
- [x] WasteCans
- [x] WasteCards

### LINKED ISSUE #6: Build related components
- [x] Time
- [x] Score
- [ ] HighScore
- [x] Credits
- [x] Background

### PR DESCRIPTION
**Pull request by ShiningElektra**
Issue #6

### CODE REVIEW

File: components/display/Display.module.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.display {
+  color: var(--primary-text);
+  font-size: var(--primary-font-size);
+  max-width: 150px;
+  min-width: 100px;
+  border-radius: 20px;
+  padding-top: 3px;
+  padding-bottom: 3px;
```
Comment (fre-ben) [inline components/display/Display.module.css]:
```suggestion
  padding: 3px 0;
```

Comment (fre-ben) [inline components/display/Display.module.css]: Try using relative units `em`

File: components/display/Display.module.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  background: linear-gradient(180deg, #0bc717 0%, #2a7903 100%);
+}
+.score {
+  text-align: left;
+  padding-left: 25px;
+}
+
+.time {
+  text-align: right;
+  padding-right: 25px;
```
Comment (fre-ben) [inline components/display/Display.module.css]: Try using relative units `em`

**Review by fre-ben — COMMENTED** [review]
LGTM 🧙

File: components/display/Display.module.css (hunk 1 above)
Comment (HorstFrank) [inline components/display/Display.module.css]:
```suggestion
  padding: 3px 0 3px 0;
```

File: components/display/Display.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  primary: boolean;
+  value: string | number;
+};
+
+function Display({ primary, value }: DisplayProps) {
+  return (
+    <div
+      className={`${styles.display} ${primary ? styles.score : ""} ${
+        !primary ? styles.time : ""
+      }`}
```
Comment (HorstFrank) [inline components/display/Display.tsx]:
```suggestion
      className={`${styles.display} ${primary ? styles.score : styles.time}`}
```

**Review by HorstFrank — COMMENTED** [review]
GJ!

File: components/game/GamePage.module.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.container {
+  /* display: grid; */
```
Comment (phlgr) [inline components/game/GamePage.module.css]: This line should be removed

File: components/game/GamePage.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import Field from "../field/Field";
+import styles from "./GamePage.module.css";
+
+const GamePage = () => {
+  return (
+    <div className={styles.container}>
+      {/* <Can /> */}
```
Comment (phlgr) [inline components/game/GamePage.tsx]: This can also be removed

File: pages/game.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        <title>Recycool</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <GamePage />
+        <form className={styles.display}>
+          <Display value="00:30" style="time" />
+          <Display value="101" style="score" />
+        </form>
```
Comment (phlgr) [inline pages/game.tsx]: Why is this a form? Does the user need to input data?

File: styles/Home.module.css [hunk 7]
```diff
@@ -1,17 +1,27 @@
...
 
-.main {
+/* .main {
   padding: 5rem 0;
   flex: 1;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
+} */
```
Comment (phlgr) [inline styles/Home.module.css:17]: Remove unused code

**Review by phlgr — CHANGES_REQUESTED** [review]
GJ 🚀

File: components/display/Display.tsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import "./Display.module.css";
+import styles from "./Display.module.css";
+
+export type DisplayProps = {
+  value: string | number | null;
```
Comment (HorstFrank) [inline components/display/Display.tsx]:
```suggestion
  value: string | number;
```

**Review by HorstFrank — COMMENTED** [review]
LGTM!!

**Review by sweisz — COMMENTED** [review]
LGTM 👍

File: pages/game.tsx (hunk 6 above)
Comment (ShiningElektra) [reply pages/game.tsx]: 🙏
No, I'll have to set a countdown in and the points should be counted.

File: components/display/Display.tsx (hunk 8 above)
Comment (ShiningElektra) [reply components/display/Display.tsx]: 🙏

**Pull request merged**